haiku-apps/qscope/qscope-1.0.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A triggered oscilloscope application"
+DESCRIPTION="QScope, a digital oscilloscope for the ADC and DAC sound streams."
+APP="QScope"
+#HOMEPAGE="https://github.com/HaikuArchives/$APP"
+HOMEPAGE="https://github.com/owenca/$APP"
+COPYRIGHT="1998 Christian Bauer"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="210bbf5ed92b0ee57e2b99f5c1c4273bab38d92c"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="f01a3a7db278291ac2084ef051025961501ff52e87aa120c3c12d37a2b64426a"
```

Comment:
```diff
 SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
 CHECKSUM_SHA256="f01a3a7db278291ac2084ef051025961501ff52e87aa120c3c12d37a2b64426a"
+SOURCE_FILENAME="$APP-$portVersion-$srcGitRev.tar.gz"
```
(It might be nice to rename the tarball to something better than just the git revision.)

Hmm, I saw that the release of the [original project](https://github.com/HaikuArchives/QScope) is 2.0, so maybe it would be nice to rename the recipe to `qscope-2.0.recipe`. Well, you would need to change the [major](https://github.com/owenca/QScope/blob/master/src/QScope.rdef#L7) number in the rdef.

BTW, I saw your changes to the original app and I think you did a great job!

Let's keep this PR open until your PR on the upstream project (https://github.com/HaikuArchives/QScope/pull/2) gets merged.
BTW, if you rename the recipe to use `2.0` as version instead of `1.0.0`, don't forget to update the rdef your created :)

Reply:
I have fixed the recipe and will push it after the other PR is merged. (I have bumped the internal version in the rdef file to 2.)

Reply:
Done.